src/pages/LoginSignupPage/LoginSignupPage.scss

```diff
@@ -0,0 +1,39 @@
+* {
+  padding: 0;
+  margin: 0;
+  overflow: hidden;
```

Comment:
why did you add `overflow: hidden;`. I think is not necessarry, We could add min-height: 100vh;